src/pages/onboarding/onboarding.jsx

```diff
@@ -0,0 +1,83 @@
+import { useEffect, useState } from "react";
+import "./onboarding.css";
+import { AiFillEdit } from "react-icons/ai";
+import { getGreet, getHour, getMinute } from "../../utils";
+import { getQuote, getWeather } from "../../services";
+import { Weather } from "../../components";
+
+export const OnBoarding = () => {
+  const [userName, setUserName] = useState("");
+  const [printUserName, setPrintUserName] = useState("");
+  const [mainFocus, setMainFocus] = useState("");
+  const [printMainFocus, setPrintMainFocus] = useState("");
+  const [hours, setHours] = useState("");
+  const [minutes, setMinutes] = useState("");
+  const [quote, setQuote] = useState("");
```

Comment:
You can make the time as a seperate component. Makes it modular.

Reply:
will do it in future